processors/zipkin/src/main/java/org/hawkular/apm/processor/zipkin/CommunicationDetailsDeriver.java

```diff
@@ -143,15 +144,14 @@ public CommunicationDetails processOneToOne(String tenantId, Span item) throws R
                 //ret.setInternal(consumer.getEndpointType() == null);
 
                 // Merge properties from consumer and producer
-                //ret.getProperties().addAll(item.getProperties());
-                //ret.getProperties().addAll(pi.getProperties());
+                List<Property> spanProperties = item.properties();
+                ret.getProperties().addAll(spanProperties);
+                ret.getProperties().addAll(pi.getProperties());
 
                 ret.setSourceFragmentId(pi.getFragmentId());
-                ret.setSourceHostName(pi.getHostName());
```

Comment:
This should still be set, even though it may currently not have a value - this code is just passing values through.